source/index.txt

```diff
@@ -53,12 +53,15 @@ describe possible configurations for a BI system.
 Hosted Database and |bi-short|
 ------------------------------
 
-`MongoDB Atlas <https://www.mongodb.com/cloud/atlas>`_ provides a fully
-hosted solution for both MongoDB and |bi-short|. Atlas dedicated
-clusters (`size M10+
-<https://docs.atlas.mongodb.com/create-new-cluster/>`_)
-offer an `integrated BI Connector instance
-<https://docs.atlas.mongodb.com/bi-connection>`_ as an option.
+.. note::
+
+   If you don’t want to manage and administer the BI Connector, you can use
```

Comment:
Manage and administer are kind of synonymous.  What if we pick one and highlight the hassle of BIC installation...

"If you don't want to install and [manage | administer] the |bi-short|, you can use..."

Reply:
Regardless of the comment above, please:

s / "BI Connector" / |bi-short|

Reply:
NIT: "If you don't want to ..."  feels so marketing-y, like we're accusing readers of being lazy.

I'm going to follow that nit up with a suggestion that I don't think is that great, what are your thoughts on:

"The |bi| is now [hosted | available] in MongoDB Atlas, our fully-managed database-as-a-service, where you don't have to install or manage your own |bi-short| instance. <rest of your blurb>"

Reply:
This last comment is a good suggestion. If it's okay, I'd like to keep the text as is (minus your other suggestions) and try suggestion in a future variant / experiment in a future week.

Reply:
Sure thing